docs/CustomMetrics.md

```diff
@@ -0,0 +1,38 @@
+How to export my own custom metrics
+=====
+Description
+-----
+Moneo provides a way for you to export your own custom node metrics. This will require you to modify/create your own custom exporter script. The [BaseExporter](./src/worker/exporters/base_exporter.py) parent class is provided for so that a child class can be created to export your custom metrics.
```

Comment:
Should we add an example use case?

Reply:
I meant for the front-end network data collection within the node_exporter to be the example case. In the instructions it says what to look at within the node_exporter

Reply:
I will mark this resolved and work on addressing this in another comment